src/types/VariableTypes.ts

```diff
@@ -33,21 +33,23 @@ export interface Variable {
     isVisible: boolean;
     isReadonly: boolean;
     isRequired: boolean;
+    occurrences: number;
 }
 
 export interface ImageVariable extends Variable {
-    occurrences: number;
     src?: ImageVariableSource;
```

Comment:
@NicolaVerbeeck Just noticed, that variable `value` field is called `src` for ImageVariable and `selected` for list variable. I think it is better to follow the same structure and call all of them `value`, isn't it?

Reply:
I agree it's not consistent. And I don't know if we want to change this now. It means migration > breaking change > WRS has to update 🥲

 For the list, though, what would be the value: items or selected?

Reply:
@Matthiee @NicolaVerbeeck @evan-mcgeek @olegsolodukhin 

Not in scope of this ticket but can I have your input on this:

Maybe indeed `ImageVariable.src` should be `ImageVariable.value` & `ListVariable.items/selected` should be `ImageVariable.value` to be consistent with the other variables.

If we agree, two thoughts comes to my mind:
- It's enough w/ the breaking-changes for now
- If we are making breaking-changes, it's better to have all of them condensed in the same time frame

WDYT?

Reply:
ListVariable.value would then be the new name for ListVariable.selected?

Reply:
for the list - value is selected. Similarity: boolean has two options - true/false, the value is the selected one; String - all possible combinations of letters (possible), the value is the "selected" one; for the list we define possible values and value refers to the chosen one.